src/pages/ach.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```

Comment:
I don't think `ach` should be a separate page 🤔 I think it's enough to enable in-context translation if we change the current language but keep the base same as english `deriv.com` if that is possible